hack/update-godep-licenses.sh

```diff
@@ -171,6 +176,8 @@ for PACKAGE in $(cat Godeps/Godeps.json | \
       file="${CONTENT[${PACKAGE}-LICENSE]-}"
   elif [[ -n "${CONTENT[${PACKAGE}-COPYRIGHT]-}" ]]; then
       file="${CONTENT[${PACKAGE}-COPYRIGHT]-}"
+  elif [[ -n "${CONTENT[${PACKAGE}-COPYING]-}" ]]; then
+      file="${CONTENT[${PACKAGE}-COPYING]-}"
   fi
   if [[ -z "${file}" ]]; then
       cat > /dev/stderr << __EOF__
```

Comment:
maybe also update the message on L187-190 to mention COPYING?

Reply:
@ixdy Thanks for the review, I have made the suggested updates.